ceci/main.py

```diff
@@ -9,44 +9,51 @@
 sys.path.append(os.getcwd())
 
 parser = argparse.ArgumentParser(description='Run a Ceci pipeline from a configuration file')
-parser.add_argument('config_filename', help='Configuration file in YAML format.')
+parser.add_argument('pipeline_config', help='Pipeline configuration file in YAML format.')
+parser.add_argument('run_config',      help='Stages configuration file in YAML format.')
```

Comment:
Here the second yaml file is always required - is that the idea? In the run function below it's optional.

Reply:
That's just me not being able to bend argparse to my will, couldn't figure out how to make a second positional argument optional :-/ 
But I realized afterwards that this is not even an issue, we can get the config file from the pipeline configuration file. So I'm removing this option.